src/content/en/tools/workbox/guides/advanced-recipes.md

```diff
@@ -0,0 +1,137 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: Advanced recipes to use with Workbox.
+
+{# wf_updated_on: 2017-12-17 #}
+{# wf_published_on: 2017-12-17 #}
+{# wf_blink_components: N/A #}
+
+# Advanced Recipes {: .page-title }
+
+{% include "web/tools/workbox/_shared/alpha.html" %}
+
+## Offer a page reload for users
```

Comment:
My gut is that we should link to https://redfin.engineering/how-to-fix-the-refresh-button-when-using-service-workers-a8e27af6df68 instead of including a recipe inline, because there are a number of different considerations and tradeoffs that that post does a good job of explaining.

If you'd prefer to include the same code inline rather than linking out then I could offer some line-by-line feedback, but I'll hold off until we decide.